packages/react-components/src/components/ChatMessage/ChatMessageComponentAsMessageBubble.tsx

```diff
@@ -338,20 +334,6 @@ const MessageBubble = (props: ChatMessageComponentAsMessageBubbleProps): JSX.Ele
             }}
             root={{
               className: chatMyMessageStyles.root,
-              onBlur: (e) => {
-                // copy behavior from North*
-                // `focused` controls is focused the whole `ChatMessage` or any of its children. When we're navigating
-                // with keyboard the focused element will be changed and there is no way to use `:focus` selector
-                const shouldPreserveFocusState = e.currentTarget.contains(e.relatedTarget);
-
-                setFocused(shouldPreserveFocusState);
-              },
-              onFocus: () => {
```

Comment:
it looks like I can't navigate to the menu button anymore with Tab and Accessibility turned on

Reply:
This looks this is something that was introduced between 27 Sept and now on main; also reproducible there.